lang/en/local_pages.php

```diff
@@ -109,4 +109,15 @@
 $string['backtolist'] = "Back to pages list";
 $string['page'] = 'Page';
 $string['form'] = 'Form';
-$string['type'] = 'Type';
\ No newline at end of file
+$string['type'] = 'Type';
+$string['email'] = "Email";
+$string['text'] = "Text";
+$string['number'] = "Number";
+$string['checkbox'] = "Checkbox";
+$string['textarea'] = "Text Area";
+$string['select'] = "Select";
+$string['html'] = "HTML";
+$string['numeric'] = "Numeric";
+$string['validemail'] = 'Please supply a valid email address for {$a->email}';
+$string['pleasefillin'] = 'Please fill in {a->name}';
+$string['pleasefillinnumber'] = 'Please give a number for {$a->name}';
```

Comment:
If the language string only has 1 placeholder then you could just use {$a}, then in get_string('validemail', 'local_pages', 'The string or a $variable that has the string value to go in here');

```suggestion
$string['validemail'] = 'Please supply a valid email address for {$a}';
$string['pleasefillin'] = 'Please fill in {$a}';
$string['pleasefillinnumber'] = 'Please give a number for {$a}';
```